src/Swift/Swift/SwiftGlueLib.cs

```diff
@@ -16,8 +16,12 @@ internal SwiftGlueLib () : base (Path)
 		{
 		}
 
-		IntPtr _bodyProtocolWitness;
-		public IntPtr BodyProtocolWitness
-			=> _bodyProtocolWitness == IntPtr.Zero ? (_bodyProtocolWitness = RequireSymbol ("$s11SwiftUIGlue9ThunkViewV4bodyq_vg")) : _bodyProtocolWitness;
+		IntPtr _viewBodyProtocolWitness;
+		public IntPtr ViewBodyProtocolWitness
+			=> _viewBodyProtocolWitness == IntPtr.Zero ? (_viewBodyProtocolWitness = RequireSymbol ("$s11SwiftUIGlue9ThunkViewV4bodyq_vg")) : _viewBodyProtocolWitness;
```

Comment:
@chkn how are you getting this Swift Symbol `s11SwiftUIGlue9ThunkViewV4bodyq_vg`, looking inside the framework?